doc/building.md

```diff
@@ -405,9 +405,6 @@ happens, read more below in [the `configure` options](#configureoptions).
 
 Some examples:
 
->  **Windows 32bit build with freetype specified:** \
```

Comment:
doc/building.html should be updated as well.

Reply:
note that the documentation for freetype in jdk8 is outdated, in jdk9 it was reworked by this [commit](https://github.com/openjdk/jdk/commit/8dc9c32d0d1bbc791555f231e51c55924f256e75). We should update at least the freetype related part as a follow-up patch.

Reply:
Yes, that commit probably needs to be backported as a starting point and then further changes made, including the other FreeType changes in the original version of this commit and replacing Mercurial with git.

I think this patch should at least update `doc/building.html` to match the `building.md` changes being made.